doc/how_to_guides/isaac_panda/.docker/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM osrf/ros:humble-desktop-jammy
+
+SHELL ["/bin/bash", "-c", "-o", "pipefail"]
+
+RUN echo "deb [trusted=yes] https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/ ./" \
```

I think these lines should be updated to match the `ROS_DISTRO`
```suggestion
RUN echo "deb [trusted=yes] https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-$ROS_DISTRO/ ./" \
```